contrib/gocql/gocql/gocql.go

```diff
@@ -111,6 +111,18 @@ func (tq *Query) newChildSpan(ctx context.Context) ddtrace.Span {
 }
 
 func (tq *Query) finishSpan(span ddtrace.Span, err error) {
+	// ErrNotFound is technically an error from gocql point of view.
```

Comment:
Can we please put this comment inside the `if` statement? Same for the one below.

Reply:
Fixed